gunicorn/config.py

```diff
@@ -1309,6 +1309,24 @@ class LogConfig(Setting):
     """
 
 
+class LogConfigDict(Setting):
+    name = "logconfig_dict"
+    section = "Logging"
+    cli = ["--log-config-dict"]
+    validator = validate_dict
+    default = {}
+    desc = """\
+    The log config dictionary to use, using the standard Python
+    logging module's dictionary configuration format. This option
+    takes precedence over the :ref:`logconfig` option, which uses the
```

Comment:
This is a minor issue, but I'd say let's deprecate `--log-config` documentation-only and tell users to switch to `--log-config-dict`.

Reply:
If you can explain a bit more, I will do a PR. Or I will review a PR. Thanks!